baseplate.go

```diff
@@ -77,17 +77,43 @@ type Server interface {
 	Serve() error
 }
 
-// Serve runs the given Server until it is given an external shutdown signal
-// using runtimebp.HandleShutdown to handle the signal and shut down the
-// server gracefully.  Returns the (possibly nil) error returned by "Close" or
+// ServeArgs provides a list of arguments to Serve.
+type ServeArgs struct {
+	// Server: Mandatory. Indicates the Server that should be run until receiving
+	// a shutdown signal.
```

Comment:
You should write comments like a normal sentence so this should be more like:

```suggestion
	// Server is the Server that should be run until receiving a shutdown signal.
	// This is a required argument and baseplate.Serve will panic if this is nil.
```